### PR TITLE
Add more systems and put some of the things back into the readme that should have stayed in last PR.

### PR DESCRIPTION
**Pull request by Trihedraf**

### CODE REVIEW

File: docs/installing.md [hunk 1]
```diff
@@ -1,19 +1,19 @@
 # Installing
 
-Download the latest [DevilutionX release](https://github.com/diasurgical/devilutionX/releases), or [build from source](./docs/building.md), and extract the contents to a location of your choosing.
+Download the latest [DevilutionX release](https://github.com/diasurgical/devilutionX/releases), or [build from source](./docs/building.md), and extract the contents to a location of your choosing or [build from source](./docs/building.md).
```
Comment (AJenbo) [inline docs/installing.md:3]: building is already in the middle of the sentience.

Comment (Trihedraf) [reply docs/installing.md:3]: derp

Comment (Trihedraf) [reply docs/installing.md:3]: Should it be at the end? Because you don't extract the contents really if you build it.

File: README.md [hunk 2]
```diff
@@ -26,7 +26,10 @@ For a full list of changes see our [changelog](docs/CHANGELOG.md).
 
 # How To Install:
 
-Download the latest [DevilutionX release](https://github.com/diasurgical/devilutionX/releases) and extract the contents to a location of your choosing.
+Download the latest [DevilutionX release](https://github.com/diasurgical/devilutionX/releases) and extract the contents to a location of your choosing or [build from source](#building-from-source).
+
+- Copy `diabdat.mpq` from your CD or GoG installation (or [extract it from the GoG installer](https://github.com/diasurgical/devilutionX/wiki/Extracting-the-.MPQs-from-the-GoG-installer)) to the folder containing the DevilutionX executable or to the data folder.
```
Comment (AJenbo) [inline README.md:31]:
```suggestion
- Copy `DIABDAT.MPQ` from your CD or GoG installation (or [extract it from the GoG installer](https://github.com/diasurgical/devilutionX/wiki/Extracting-the-.MPQs-from-the-GoG-installer)) to the folder containing the DevilutionX executable or to the data folder.
```

File: README.md [hunk 3]
```diff
@@ -26,7 +26,10 @@ For a full list of changes see our [changelog](docs/CHANGELOG.md).
...
+- To run the Hellfire expansion of Diablo you will need to copy `hellfire.mpq`, `hfmonk.mpq`, `hfmusic.mpq`, `hfvoice.mpq` into the same location as diabdat.mpq.
```
Comment (AJenbo) [inline README.md:32]:
```suggestion
- To run the Hellfire expansion of Diablo you will need to copy `hellfire.mpq`, `hfmonk.mpq`, `hfmusic.mpq`, `hfvoice.mpq` into the same location as `DIABDAT.MPQ`.
```

**Pull request merged**